src/paste-keyboard-shortcut.ts

```diff
@@ -0,0 +1,26 @@
+type UMap = WeakMap<HTMLElement, boolean>
+export class PasteKeyBoardShortcut {
```

Comment:
should this class (and the file name, and the instances) be something like `PasteFormatSkipper`?